tests/hit/test_build_hit.py

```diff
@@ -98,6 +98,33 @@ def test_outputs_specification(dsp_test_file, tmptestdir):
     assert list(obj.keys()) == ["calE", "AoE", "A_max"]
 
 
+def test_aggregation_outputs(dsp_test_file, tmptestdir):
+    outfile = f"{tmptestdir}/LDQTA_r117_20200110T105115Z_cal_geds_hit.lh5"
+
+    build_hit(
+        dsp_test_file,
+        outfile=outfile,
+        hit_config=f"{config_dir}/aggregations-hit-config.json",
+        wo_mode="overwrite",
+    )
+
+    sto = LH5Store()
+    obj, _ = sto.read_object("/geds/hit", outfile)
+    assert list(obj.keys()) == [
+        "is_valid_rt",
+        "is_valid_t0",
+        "is_valid_tmax",
+        "aggr1",
+        "aggr2",
+    ]
+
+    df = store.load_dfs(outfile, ["aggr1", "aggr2"], "geds/hit/")
+
+    # aggr1 consists of 3 bits --> max number can be 7, aggr2 consists of 2 bits so max number can be 3
+    assert not (df["aggr1"] > 7).any()
+    assert not (df["aggr2"] > 3).any()
+
```

Comment:
Can you add a test that checks whether each bit has the expected value? With this function, for example:
```py
def get_bit(x, n):
    return x & 1 << (n - 1) != 0
```
where `n` is the bit number (from least to most significant) and numbering starts from 1.

Reply:
Lovely thank you!

Reply:
I've added this test